docs/faq.md

```diff
@@ -1,41 +1,57 @@
 
-A: The easiest place to have the `ansible.cfg` is in the project directory adjacent to the playbook. The playbook directory is automatically mounted in the execution environment and the `ansible.cfg` file will be found.  If the `ansible.cfg` file is in another directory, the `ANSIBLE_CONFIG` variable needs to be set and the directory specified as a custom volume mount. (See the [settings guide](settings.rst) for `execution-environment-volume-mounts`)
+## Placement of ansible collections
 
-**Q: Where should the `ansible.cfg` file go when not using an execution environment?**
+### Where should ansible collections be placed when using an execution environment?
 
-A: ansible will look for the `ansible.cfg` in the typical locations when not using an execution-environment.  (See the ansible docs for the possibilities)
+The easiest place to have ansible collections is in the project directory, in a playbook adjacent collections directory. (eg `ansible-galaxy collections install ansible.utils -p ./collections`).  The playbook directory is automatically mounted in the execution environment and the collections should be found. Another option is to build the collections into an execution environment using [ansible builder](https://ansible-builder.readthedocs.io/en/latest/). This was done to help playbook developers author playbooks that are production ready, as both ansible controller and awx support playbook adjacent collection directories. If the collections are in another directory, the `ANSIBLE_COLLECTIONS_PATHS` variable needs to be set and the directory specified as a custom volume mount. (See the [settings guide](settings.rst) for `execution-environment-volume-mounts`)
 
-**Q: Where should ansible collections be placed when using an execution environment?**
+### Where should ansible collections be placed when not using an execution environment?
 
-A: The easiest place to have ansible collections is in the project directory, in a playbook adjacent collections directory. (eg `ansible-galaxy collections install ansible.utils -p ./collections`).  The playbook directory is automatically mounted in the execution environment and the collections should be found. Another option is to build the collections into an execution environment using [ansible builder](https://ansible-builder.readthedocs.io/en/latest/). This was done to help playbook developers author playbooks that are production ready, as both ansible controller and awx support playbook adjacent collection directories. If the collections are in another directory, the `ANSIBLE_COLLECTIONS_PATHS` variable needs to be set and the directory specified as a custom volume mount. (See the [settings guide](settings.rst) for `execution-environment-volume-mounts`)
+When not using an execution environment, ansible will look in the default locations for collections.  For more information about these, check out the [collections guide](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html).
```

Comment:
Links like this (to other Sphinx-managed sites) should eventually make use of intersphinx instead of hardcoded URLs.